### DISCUSSION TITLE
GLPI agent MacOSX documentation question

### DISCUSSION THREAD

**Discussion by crunchy59** [glpi-project/glpi-agent#288] [General]
Hello, in this link 
https://glpi-agent.readthedocs.io/en/latest/installation/index.html
for mac os x, is not local.cfg file for configuration, is glpi.cfg

Cheers.
C

**Comment by g-bougard** [comment]
Hi @crunchy59 

by default, on MacOSX, all files under `/Applications/GLPI-Agent/etc/conf.d` folder only needs to have the `.cfg` extension. Indeed it only depends on what is set as `include` directive in the `/Applications/GLPI-Agent/etc/agent.cfg`. So the file basename itself is not important. It may be important only if a configuration is set in different files under "conf.d". And in that case, you should know files are read in alphabetic order and so the file read will really set a parameter defined in more than one file.

So in the doc, "local.cfg" is just a file example. Personnaly, I'm using "teclib.cfg" as filename in "conf.d" subfolder.
